scripts/exglobal_atmos_analysis.sh

```diff
@@ -411,6 +412,49 @@ fi
 
 cd $DATA || exit 99
 
+<<<<<<< HEAD:scripts/exglobal_analysis_fv3gfs.sh.ecf
```

Comment:
Please remove the conflict

Reply:
This still needs to be cleaned up.  Please remove the <<<<HEAD section.

Reply:
cleaned up